src/docfx/build/toc/TableOfContentsMap.cs

```diff
@@ -218,7 +221,7 @@ where tocToTocs.ContainsKey(item.Key) && !item.Key.IsExperimental()
 
         private void SplitToc(FilePath file, TableOfContentsNode toc, ConcurrentBag<FilePath> result)
         {
-            if (string.IsNullOrEmpty(toc.SplitItemsBy) || toc.Items.Count <= 0)
+            if (!Array.Exists(_config.SplitTOC, e => e.Equals(file.Path.Value)) || toc.Items.Count <= 0)
```

Comment:
should `file.Path.Value` be case insensitive?

Reply:
No.